package.json

```diff
@@ -107,6 +107,7 @@
     "mkdirp": "0.5.1",
     "mocha": "3.1.2",
     "nock": "8.0.0",
+    "node-jose": "1.0.0",
```

Comment:
As mentioned below, taking this dependency just to include crypto in the tests doesn't seem worthwhile, not when the auth-server doesn't notice or enforce any of the crypto details.